platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -190,20 +190,21 @@ This will allow Gradle to perform additional performance optimizations and will
 
 While the existing callbacks continue to work, we encourage everyone to adopt the new API and provide us with early feedback.
 
-The example below shows how this new API could be used in a settings script or [settings plugins](userguide/custom_plugins.html#project_vs_settings_vs_init_plugins):
+The example below shows how this new API could be used in a settings script or [settings plugins](userguide/custom_plugins.html#project_vs_settings_vs_init_plugins) to apply configuration to all subprojects, 
+while avoiding [cross-project configuration](userguide/sharing_build_logic_between_subprojects.html#sec:convention_plugins_vs_cross_configuration):
 
-```groovy
-// settings.gradle
-rootProject.name = 'root'
+```kotlin
+// settings.gradle.kts
+include("sub1")
+include("sub2")
 
-def start = 0
 gradle.lifecycle.beforeProject {
-    start = System.currentTimeMillis()
-}
-gradle.lifecycle.afterProject { project ->
-    def end = System.currentTimeMillis()
-    def elapsed = end - start
-    println "Project $project.name configured in ${elapsed}ms"
+    if (this.path != this.rootProject.path) {
```

Comment:
❓ Do we need to complicate the example with a conditional here? I'd just keep the body of the `if`, but if you feel strongly about showing it's possible to read project properties I'd ditch the `this.` and go for the idiomatic `if (parent != null)`

Reply:
Not really, will get rid of it, and change the description